AEPCore/Sources/rules/LaunchRulesEngine.swift

```diff
@@ -30,31 +30,26 @@ class LaunchRulesEngine {
     private let transform: Transforming
     private let name: String
     private let extensionRuntime: ExtensionRuntime
-    private let rulesQueue = DispatchQueue(label: "com.adobe.launch.rulesengine.process")
-    private var cachedEvents: [Event]?
+    private let rulesQueue: DispatchQueue
+    private var waitingEvents: [Event]?
     private let dataStore: NamedCollectionDataStore
 
     let evaluator: ConditionEvaluator
     let rulesEngine: RulesEngine<LaunchRule>
-    let rulesDownloader: RulesDownloader
 
-    init(name: String, extensionRuntime: ExtensionRuntime) {
+    public init(name: String, extensionRuntime: ExtensionRuntime) {
```

Comment:
would be good to get some in-code documentation here, especially for public apis